pkg/network/protocols/kafka/telemetry_test.go

```diff
@@ -88,16 +88,16 @@ func TestTelemetry_Count(t *testing.T) {
 
 func verifyHitsCount(t *testing.T, telemetry *Telemetry, tx *KafkaTransaction) {
 	if tx.Request_api_key == 0 {
-		if tx.Request_api_version < MinSupportedProduceRequestApiVersion || tx.Request_api_version > MaxSupportedProduceRequestApiVersion {
+		if tx.Request_api_version < ClassificationMinSupportedProduceRequestApiVersion || tx.Request_api_version > DecodingMaxSupportedProduceRequestApiVersion {
```

Comment:
the min is from `Classification` the max from `Decoding`
is there a reason for that?